ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -17,8 +17,8 @@ gentoo_prefix_path: /cvmfs/{{ cvmfs_repository }}/versions/{{ eessi_version }}/c
 
 # How to build the prefix
 prefix_required_space: 15 GB
-prefix_snapshot_url: https://eessi-gentoo-snapshot.s3-eu-west-1.amazonaws.com
-prefix_snapshot_version: 20211120
+prefix_gentoo_overlay: https://github.com/gentoo/gentoo
```

Comment:
Hmm, perhaps `overlay` is misleading here (I first though this was about our `EESSI/gentoo-overlay` repo)?

Reply:
Just `gentoo_git_repo` maybe?

Reply:
Okay, renamed it to `gentoo_git_repo` (and `gentoo_git_commit`) in https://github.com/EESSI/compatibility-layer/pull/143/commits/e767711a2c881ae86f5970dc5d8b652be8b2d00e.